app/magento2/base.yml

```diff
@@ -78,10 +86,15 @@ app:
         stdout: true
         stderr: true
       config:
+        nginx_conf_main_entry:
+          from: assets/magento-nginx-entrypoint.conf
+          to: '{{ get_working_directory() }}/nginx.conf.main_entry'
         env_php:
           from: assets/env.php
           to: '{{ get_working_directory() }}/app/etc/env.php'
       pre_start:
+        # Remove main entrypoint from nginx.conf.sample
+        - "sed -i'' -z -E 's/# PHP entry point for main application(\n|.)*}\n\n//' '{{ get_working_directory() }}/nginx.conf.sample'"
```

Comment:
This isn't portable, it doesn't work in CentOS 7 based images:
```
/entrypoint_riptide.sh: eval: line 219: syntax error near unexpected token `(\'\n/entrypoint_riptide.sh: eval: line 219: `exec su riptide -m -c \' /bin/sh -c "sed -i\'\' -z -E \'s/# PHP entry point for main application( |.)*} //\' \'/src/www/nginx.conf.sample\'" /usr/local/bin/docker-entrypoint.sh \'\'\n
```

Additionally this changes the versioned nginx.conf.sample in projects. We need a different approach.